_topic_map.yml

```diff
@@ -1103,12 +1103,15 @@ Topics:
     File: image-pull-policy
   - Name: Using image pull secrets
     File: using-image-pull-secrets
-- Name: Managing imagestreams
+- Name: Managing image streams
```

Comment:
We have resource named `imagestream`, but not `image stream`.IMO it should be `imagestream`. 
@dmage Please correct me if I am wrong, thanks

Reply:
Thank you! This came up in OSDOCS-999 too. We have new guidance (https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting) that changes the way we're writing "imagestream" to "image stream" with general references. Is that something that is going to cause trouble for users? If so, I can reach out and see if we can adjust the guidance.

Reply:
I haven't heard a complaint about it. If so, I agreed this change.
Thanks!